php/ice/config.yaml

```diff
@@ -0,0 +1,23 @@
+framework:
+  website: iceframework.org
+  version: 1.5.1
+
+build_deps:
+  - bash
+
+before_build:
+  - wget -O /tmp/framework-1.5.1.tar.gz https://github.com/ice/framework/archive/1.5.1.tar.gz
```

Comment:
could ice be installed with `pecl` or some other tool ?

Reply:
There is a packege, but only for the openSUSE linux https://software.opensuse.org/package/php7-ice, so it's needed to compile.

Reply:
In fact, I thought about a `pecl`, so as to have a tool doing compilation stuff fo us.
However, it is ok for this project. Do you have considered to distribute`ice` with `pecl` (it coul be easier when this project will run on cloud)

Reply:
Agree. I will try to add to the pecl repository.